src/server/redis_connection.cc

```diff
@@ -130,6 +130,39 @@ void Connection::Reply(const std::string &msg) {
   redis::Reply(bufferevent_get_output(bev_), msg);
 }
 
+std::string Connection::Bool(bool b) const {
+  if (protocol_version_ == RESP::v3) {
+    return b ? "#t" CRLF : "#f" CRLF;
+  }
+  return Integer(b ? 1 : 0);
+}
+
+std::string Connection::MultiBulkString(const std::vector<std::string> &values,
+                                        bool output_nil_for_empty_string) const {
```

Comment:
Is there any scenario that `output_nil_for_empty_string` is false?

Reply:
Ahh I see it is used in Array2RESP. But if it is just used here maybe we can remove it.

Reply:
Yes, it will set to false if we don't want to convert the empty string to nil. For example, in SMEMBERS command we don't want to convert the empty member string to nil.

Reply:
I see. Thank you.